packages/menu-bar/src/vaadin-menu-bar-interactions-mixin.js

```diff
@@ -94,17 +127,32 @@ export const InteractionsMixin = (superClass) =>
       if (target) {
         this._buttons.forEach((btn) => {
           this._setTabindex(btn, btn === target);
+          if (btn === target && btn !== this._overflow) {
+            this._showTooltip(btn);
```

Comment:
There is a problem with `focusin` as the tooltip does not become hidden on `mousedown` when the menu-bar button gets focus, because in this case `mousedown` happens before `focusin`:

https://user-images.githubusercontent.com/10589913/188824816-49000f6c-0d9b-48ea-85fe-64b76881a40d.mp4

Let's add a check for `keyboardActive` flag to only show tooltip on keyboard focus.